src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs

```diff
@@ -120,6 +142,35 @@ bool Execute (DirectoryAssemblyResolver resolver)
 		readonly List<string> do_not_package_atts = new List<string> ();
 		int indent = 2;
 
+		AssemblyDefinition ResolveRuntimeAssemblyForReferenceAssembly (LockFile lockFile, DirectoryAssemblyResolver resolver, AssemblyNameDefinition assemblyNameDefinition)
```

Comment:
I'm assuming this method was cribbed from .NET Core? If that's the case, we should likewise mention that in `src/Xamarin.Android.Build.Tasks.tpnitems`.

Reply:
nope, it was written by me. I referenced the Nuget source code to see how the API Nuget.ProjectModel was used but none of this code was copied from .Net Core or Nuget.